spec/api.yaml

```diff
@@ -124,6 +124,150 @@ paths:
               schema:
                 $ref: '#/components/schemas/User'
 
+  /user/auth/forgot-password:
+    post:
+      summary: 'Send Forgot password Email OTP'
+      tags:
+        - onboarding
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ForgotPasswordRequestDto'
+      responses:
+          200:
+            description: 'Success'
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/DefaultSuccessResponse200'
+          404:
+            description: 'Not found Request'
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/ErrorResponse404'
+
+
+  /user/auth/otp-verify/forgot-password:
+    post:
+      summary: 'verify password reset otp'
+      tags:
+       - onboarding
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/verifyForgetPasswordPRequestDto'
```

Comment:
`verifyForgetPasswordPRequestDto` start with Capital letter, Remove the P between Password and Request